pandas/tests/frame/test_arithmetic.py

```diff
@@ -1882,3 +1882,19 @@ def _constructor_sliced(self):
 
     result = sdf + sdf
     tm.assert_frame_equal(result, expected)
+
+
+@pytest.mark.parametrize("level", [0, None])
+def test_broadcast_multiindex(level):
```

Comment:
@jbrockmendel test in a good location?

Reply:
id put it in the TestFrameFlexArithmetic class in this file